File: 12015.py
from sys import stdin

def main():
	casos=int(input())
	urls=[]
	relevancias=[]
	for i in range(casos):
		valor=0
		urls=[]
		relevancias=[]
		for k in range(10):
			line=list(map(str,stdin.readline().strip().split()))
			url=str(line[0])
			relevancia=int(line[1])
			urls.append(url)
			relevancias.append(relevancia)
		
		for j in range(10):
			if(relevancias[j]>=valor):
				valor=relevancias[j]

		print("Case #"+str(i+1)+":")
		for t in range(10):
			if(relevancias[t]==valor):
				print(urls[t])
main()

File: 12015.c
#include<stdio.h>
#include<stdlib.h>
#include<string.h>

int main(){
	int casos,relevancia[101],i=0,j=0, h=0,max,k=0;
	char url[105][105];
	scanf("%d",&casos);
	while(casos--){
		i = 0;
		j = 0;
		h = 0;
		max = 0;
		while(i < 10){
			scanf("%s %d/n",url[i],&relevancia[i]);
			i++;
		}
		while(j < 10){
			if(relevancia[j]>=max){
				max=relevancia[j];
			}
			j++;
		}
		printf("Case #%d:\n",++k);
		while(h<10){
			if(relevancia[h]==max){
				printf("%s\n",url[h]);
			}
			h++;
		}
	}
}
